tests/fixtures/get-cli-test-fixture.ts

```diff
@@ -32,14 +63,28 @@ export async function getCliTestFixture(): Promise<CliTestFixture> {
     return { stdout, stderr }
   }
 
+  // Setup cleanup
   const cleanup = async () => {
     await rm(tmpDir, { recursive: true, force: true })
+    if (server && typeof server.stop === "function") {
+      await server.stop()
+    }
+    cliConfig.clear()
   }
 
-  afterEach(cleanup)
+  afterEach(async () => {
+    // Reset database between tests if needed
+    // This is optional - you might want to keep state between tests
+  })
+  
+  afterAll(cleanup)
 
   return {
     tmpDir,
     runCommand,
+    server,
```

Comment:
Make sure to call this registryServer, all of these should have a registry prefix. The cli also runs a server and we dont want to confuse them